tests/cases/notification-generator-sw.js

```diff
@@ -20,6 +20,9 @@ self.addEventListener('activate', function(event) {
 self.addEventListener('fetch', function(event) {});
 
 self.addEventListener('notificationclick', function(event) {
+  if (event.reply) {
+     alert('User replied: "' + event.reply + '"');
+  }
```

Comment:
You'll want to add this around line 59, currently it doesn't do anything.

Reply:
Ah I see, so you can't show an alert from a service worker, but you can from a page, hence the postMessage business..

I'll also add instructions to the test survey to select the 'Display an alert' reaction setting then.

Reply:
(Done)